src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -25,12 +26,31 @@ namespace OpenTelemetry.Exporter
     public sealed class ZipkinExporterOptions
     {
         internal const int DefaultMaxPayloadSizeInBytes = 4096;
+        internal const string ZipkinEndpointEnvVar = "OTEL_EXPORTER_ZIPKIN_ENDPOINT";
+        internal const string DefaultZipkinEndpoint = "http://localhost:9411/api/v2/spans";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZipkinExporterOptions"/> class.
+        /// Initializes zipkin endpoint.
+        /// </summary>
+        public ZipkinExporterOptions()
+        {
+            try
+            {
+                this.Endpoint = new Uri(Environment.GetEnvironmentVariable(ZipkinEndpointEnvVar) ?? DefaultZipkinEndpoint);
+            }
+            catch (Exception ex)
```

Comment:
minor (may follow up in separate PR):
We can do similar to what Jaeger/OTLP does - catch SecurityException for reading env variable. Handle the Uri parse exception separately. Both should be logged as separate EventSource events.
This avoids catching the generic Exception.

Reply:
Sure - will create a separate PR for that.